### DISCUSSION TITLE
Author box for articles

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#307] [Ideas]
Hi @razonyang 

An author box inside the post is a great way for visitors to know a bit more about who wrote the article they are reading. This is very common in news websites that have multiple authors.

For example, the [Blowfish theme](https://n9o.xyz/posts/202403-execution-is-king/) has the author box in the beginning of the post, while the [Mainroad theme](https://mainroad-demo.netlify.app/post/hugo-template-primer/) has it at the end.

This feature isn't a must have, but I think it's very neat.  This isn't a priority, so feel free to refuse it or introduce it later down the road.

Thanks again for your awesome theme!

**Comment by razonyang** [comment]
Implementation is easy, but it seems dulplicate to the one inside post meta.

![image](https://github.com/hbstack/theme-cards/assets/17720932/41272077-2422-4a87-b527-dbbc8ffb12fd)

**Reply by LITUATUI** [reply]
Yes, that's true. 

But I think the author deserves more visibility with an author box at the end of the article (that includes picture, description, social media and maybe even a donation link).

As I said, it's not a priority, but definitely a nice feature to have.

**Reply by razonyang** [reply]
> author deserves more visibility.

Sounds reasonable, it would help branding/promoting themself, tracked this feature on hbstack/blog#790, wont happen fast.

**Reply by razonyang** [reply]
See hbstack/blog#793 for details.
